src/toil/batchSystems/torque.py

```diff
@@ -196,25 +195,12 @@ def generateTorqueWrapper(self, command, jobID):
             stdoutfile: str = self.boss.formatStdOutErrPath(jobID, r'${PBS_JOBID}', 'out')
             stderrfile: str = self.boss.formatStdOutErrPath(jobID, r'${PBS_JOBID}', 'err')
 
-            _, tmpFile = tempfile.mkstemp(suffix='.sh', prefix='torque_wrapper')
-            fh = open(tmpFile , 'w')
-            fh.write("#!/bin/sh\n")
-            fh.write("#PBS -o {}\n".format(stdoutfile))
-            fh.write("#PBS -e {}\n".format(stderrfile))
-            fh.write("cd $PBS_O_WORKDIR\n\n")
-            fh.write(command + "\n")
-
-            fh.close
-
-            return tmpFile
-
-
-    @classmethod
-    def obtainSystemConstants(cls):
-
-        # See: https://github.com/BD2KGenomics/toil/pull/1617#issuecomment-293525747
-        logger.debug("PBS/Torque does not need obtainSystemConstants to assess global cluster resources.")
-
+            _, tmp_file = tempfile.mkstemp(suffix='.sh', prefix='torque_wrapper')
+            with open(tmp_file, 'w') as f:
+                f.write("#!/bin/sh\n")
+                f.write("#PBS -o {}\n".format(stdoutfile))
+                f.write("#PBS -e {}\n".format(stderrfile))
+                f.write("cd $PBS_O_WORKDIR\n\n")
+                f.write(command + "\n")
```

Comment:
Is this missing `return tmpFile`?

Reply:
Nope, it's on [line 206](https://github.com/DataBiosphere/toil/pull/3529/files#diff-33bd8eda6e64d400c55cf8043aa23ce884ce8147caf20df1e4a674aa975a9135R206).